### PR TITLE
Update grunt.md

### PR DESCRIPTION
**Pull request by wfhutch**

### CODE REVIEW

**Comment by lynnsamuelson** [comment]
what about grunt plugins? Steve's curriculum #43 https://github.com/nashville-software-school/front-end-curriculum/tree/master/43-compile-grunt

File: front-end-notes/grunt.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+module.exports = function(grunt) {
+
+ grunt.initConfig({
+   jshint: {
+     files: ['./javascripts/**/*.js']
+   },
+ });
+
+ require('matchdep').filterDev('grunt-*').forEach(grunt.loadNpmTasks);
+ grunt.registerTask('default', ['jshint', 'watch']);
```
Comment (NewEvolution) [inline front-end-notes/grunt.md]: Grunt will not run as is, as you have a task _watch_ in the _grunt.registerTask_ but no task named _watch_ in the task list, only _jshint_

Comment (C-Stein) [inline front-end-notes/grunt.md]: Could be useful to include a link to Steve's lots-of-Grunt-plugins (https://github.com/nashville-software-school/front-end-curriculum/tree/master/43-compile-grunt)

Comment (wfhutch) [inline front-end-notes/grunt.md]: Thanks for pointing those things out!

**Pull request merged**